doc/release/release_dev.rst

```diff
@@ -22,25 +21,119 @@ X contributors. Highlights include:
 
 - Dropped support for Python 3.6.
 - NumPy, SciPy, Matplotlib, and pandas are now default requirements.
+- Improved example gallery
+- Removed code for supporting Jython/IronPython
+- The ``__str__`` method for graph objects is more informative and concise.
+- Improved import time
+- Improved test coverage
+- New documentation theme
+- Add functionality for drawing self-loop edges
+- Add approximation algorithms for Traveling Salesman Problem
+
+New functions:
+
+- Panther algorithm
+- maximum cut heuristics
+- equivalence_classes
+- dedensification
+- random_ordered_tree
+- forest_str
+- snap_aggregation
+- networkx.approximation.diameter
+- partition_quality
+- prominent_group
+- prefix_tree_recursive
+- topological_generations
+
+NXEPs
+-----
+
+**N**\etwork\ **X** **E**\nhancement **P**\roposals capture changes
+that are larger in scope than typical pull requests, such as changes to
+fundamental data structures.
+The following proposals have come under consideration since the previous
+release:
+
+- :ref:`NXEP2` API design of view slices
```

Comment:
`API design of view slices` is redundant in the render docs, since ":ref:`NXEP2`" already expands to include the title.